test/Engine/ProtoTest/Associative/MicroFeatureTests.cs

```diff
@@ -3169,6 +3169,21 @@ public void TestNullsOnExpression01()
             thisTest.Verify("b", false);
         }
 
+        [Test]
+        public void TestUndefinedTypedIdentifier()
+        {
+            string code =
+@"
+        a : UndefinedType;
+        b : UndefinedType2 = 2;
+        c : UndefinedType3 = null;
+";
+            thisTest.RunScriptSource(code);
+            thisTest.Verify("a", null);
+            thisTest.Verify("b", null);
+            thisTest.Verify("c", null);
 
+            thisTest.VerifyRuntimeWarningCount(3);
+        }

```

Comment:
git diff sucks.
Can you just comment here the line of code you added in the testcase :)